chall02/gconde-m.py

```diff
@@ -0,0 +1,57 @@
+from argparse import ArgumentParser
+import sys
+import os
+
+morse = {
+'a': '-',
+'b':'-...', 
+'c':'-.-.', 
+'d':'-..', 
+'e':'.', 
+'f':'..-.', 
+'g':'--.', 
+'h':'....', 
+'i':'..', 
+'j':'.---', 
+'k':'-.-', 
+'l':'.-..', 
+'m':'--', 
+'n':'-.', 
+'o':'---', 
+'p':'.--.', 
+'q':'--.-', 
+'r':'.-.', 
+'s':'...', 
+'t':'-', 
+'u':'..-', 
+'v':'...-', 
+'w':'.--', 
+'x':'-..-', 
+'y':'-.--', 
+'z':'--..',
+' ':' '
+}
+
+str = ''
+i = 0;
+y = 0;
+if len(sys.argv) != 2 or not sys.argv[1]:
+    print("<a-zA-Z string>")
+    exit()
+else :
+    while i < len(sys.argv[1]):
```

Comment:
In Python you should use 
```python
for elem in tab
```

more than use indexes. As you did your code, it seems like a C program. You should read more about Python and practice